internal/ethapi/api.go

```diff
@@ -1010,6 +1010,59 @@ func (s *BlockChainAPI) GetBlockReceipts(ctx context.Context, blockNrOrHash rpc.
 	return result, nil
 }
 
+func (s *BlockChainAPI) GetBlobSidecars(ctx context.Context, blockNrOrHash rpc.BlockNumberOrHash) ([]map[string]interface{}, error) {
+	block, err := s.b.BlockByNumberOrHash(ctx, blockNrOrHash)
+	if block == nil || err != nil {
+		// When the block doesn't exist, the RPC method should return JSON null
+		// as per specification.
+		return nil, nil
```

Comment:
return nil, nil-->return nil, err
don't swallow the error

Reply:
it is ok to follow the definition of `GetBlockReceipts()`, when failed to get Block, just return nil, nil